cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js

```diff
@@ -80,7 +80,7 @@ export default function FileTable({ items }) {
                   </Button>
                 )}
                 <Button as="a" icon size="mini" primary {...downloadProp}>
-                  <Icon name="download" /> Download
```

Comment:
The button label should be dynamic:

- for data files attached via file indexes, such as https://opendata.cern.ch/record/1, the button should say "Download index";

- for data files attached directly, such as https://opendata.cern.ch/record/5500, the button should say "Download".

(The index files can be recognised by having `files.type` field value "index.json" or "index.txt" in the record JSON.)

Reply:
@psaiz The dynamic button label would be indeed needed: we have had to explain this separately to the OD users as they got confused. Now we will have more files that are small enough to be downloaded so this is even more important.